website/content/docs/auth/gcp.mdx

```diff
@@ -363,3 +363,132 @@ The GCP Auth Plugin has a full HTTP API. Please see the
 [identity-group-aliases]: /api/secret/identity/group-alias
 [instance-identity]: https://cloud.google.com/compute/docs/instances/verifying-instance-identity
 [repo]: https://github.com/hashicorp/vault-plugin-auth-gcp
+
+## Code Example
+
+The following code snippet demonstrates Google Cloud auth method to authenticate
```

Comment:
Change it to "demonstrates **the** Google Cloud auth method" and I think we're good to go!